ansible/roles/nginx/tasks/gcs.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name:  install necessary packages for Gcloud
+  apt:
+    name:
+      - apt-transport-https
+      - ca-certificates
+      - gnupg
```

Comment:
@zhquan do you think these packages should be available also for the `common` role?

Reply:
Yes it makes sense, we should move it to the `common` role

Reply:
These packages are already in the common role so its not needed